scripts/create-kubestellar-demo-env.sh

```diff
@@ -17,16 +17,27 @@
 
 set -e
 
-kubestellar_version=0.25.0-rc.1
+export kubestellar_version=0.25.0-rc.1
+echo -e "KubeStellar Version: ${kubestellar_version}"
 
 echo -e "Checking that pre-req softwares are installed..."
 curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -V kflex ocm helm kubectl docker kind
 
-output=$(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -A -V kflex ocm helm kubectl docker kind
-)
+output=$(curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${kubestellar_version}/hack/check_pre_req.sh | bash -s -- -A -V kflex ocm helm kubectl docker kind)
```

Comment:
Why is this curl-to-bash being done a second time? Nothing is done with the content of `output`, right?

Reply:
removed